src/conversions.jl

```diff
@@ -19,6 +19,11 @@ function SparseMatrixCSC(M::PermMatrix)
     SparseMatrixCSC(n, n, collect(1:n+1), order, M.vals[order])
 end
 
+function SparseMatrixCSC(M::Diagonal)
+    n = size(M, 1)
+    SparseMatrixCSC(n, n, collect(1:n+1), collect(1:n), M.diag)
```

Comment:
seems this line is using the wrong args.

Reply:
What is the correct one?